test/tools/TestExe/TestExe.cs

```diff
@@ -18,6 +19,9 @@ private static int Main(string[] args)
                     case "-echoargs":
                         EchoArgs(args);
                         break;
+                    case "-echoraw":
```

Comment:
Should this be `-echocommandline`?  "echoraw" implies something specific to echo

Reply:
Changed to `-echocmdline`, and make it only output the raw command line received by the process (on windows).